feat(docs)add Logs Ingest + Logs Query manage-usage pages (#46095)
supabase/supabase: apps/docs/content/guides/platform/manage-your-usage/logs-ingest.mdx

| Line Item           | Units     | Costs                    |
| ------------------- | --------- | ------------------------ |
| Pro Plan            | 1         | <Price price="25" />     |
| Compute Hours Micro | 744 hours | <Price price="10" />     |
| Logs Ingest         | 2 GB      | <Price price="0" />      |
| **Subtotal**        |           | **<Price price="35" />** |
| Compute Credits     |           | -<Price price="10" />    |
| **Total**           |           | **<Price price="25" />** |

### Exceeding quota

The organization's Logs Ingest usage exceeds the quota by 7 GB, incurring charges for this additional usage.

| Line Item           | Units     | Costs                      |
| ------------------- | --------- | -------------------------- |
| Pro Plan            | 1         | <Price price="25" />       |
| Compute Hours Micro | 744 hours | <Price price="10" />       |
| Logs Ingest         | 12 GB     | <Price price="3.5" />      |
| **Subtotal**        |           | **<Price price="38.5" />** |
| Compute Credits     |           | -<Price price="10" />      |
| **Total**           |           | **<Price price="28.5" />** |

## View usage

> You can view Logs Ingest usage on the [organization's usage page](/dashboard/org/_/usage). The page shows the usage of all projects by default. To view the usage for a specific project, select it from the dropdown. You can also select a different time period.

Reviewer: ```suggestion
You can view Logs Ingest usage on the [organization's usage page](/dashboard/org/_/usage) of the Dashboard. The page shows the usage of all projects by default. To view the usage for a specific project, select it from the dropdown. You can also select a different time period.
```

This is an attempt to be explicit about Dashboard links to maybe help with AI tools…

{/* TODO: Add screenshots once Studio surfaces are live */}

## Optimize usage

> Logs are generated automatically by every service in your Supabase project — you don't write them directly. Log volume scales with your application's traffic and behavior. To reduce ingest volume:

Reviewer: ```suggestion
Every service in your Supabase project automatically generates Logs — you don't write them directly. Log volume scales with your application's traffic and behavior. To reduce ingest volume:
```

- **Reduce log-level verbosity** in your Edge Functions and server-side code (for example, `info` → `warn` in production).
- **Audit verbose logging in your application code.** Application-level logs forwarded to Supabase services count toward ingest.
- **Cap log payload size.** Large structured payloads inflate GB-billed volume quickly.

> - **Investigate spikes.** Use the [Logs Explorer](/dashboard/project/_/logs-explorer) to find services or endpoints producing unusually high volume.

Reviewer: ```suggestion
- **Investigate spikes.** Use the [**Logs Explorer**](/dashboard/project/_/logs-explorer) section of the Dashboard to find services or endpoints producing unusually high volume.
```
